users/src/main/java/io/crate/http/netty/HttpAuthUpstreamHandler.java

```diff
@@ -98,6 +97,9 @@ private void handleHttpChunk(ChannelHandlerContext ctx, MessageEvent e) {
 
     @VisibleForTesting
     static void sendUnauthorized(Channel channel, @Nullable String body) {
+        if (LOGGER.isWarnEnabled() && body != null) {
```

Comment:
warn condition also not needed
what does this mean when body is null? might be interesting too?